packages/react-query/src/useSuspenseQueries.ts

```diff
@@ -31,11 +31,11 @@ interface ToInferWithEnabledOption<TQueryFnData, TQueryKey extends QueryKey, TEn
 }
 interface ToInferWithSelectOption<TQueryFnData, TData, TQueryKey extends QueryKey>
   extends ToInferOption<TQueryFnData, TQueryKey> {
-  select: (data: unknown) => TData
+  select: (data: any) => TData
```

Comment:
In my opinion, we should see useQueries implementation of TanStack React Query **v4**. because @suspensive/react-query target tanstack query v4. and they used any too. so we follow it.

https://github.com/TanStack/query/blob/v4/packages/react-query/src/useQueries.ts#L57